lib/Db/FaceMapper.php

```diff
@@ -197,5 +188,6 @@ public function insertFace(Face $face, IDBConnection $db = null) {
 				'creation_time' => $qb->createNamedParameter($face->creationTime, IQueryBuilder::PARAM_DATE),
 			])
 			->execute();
+		$face->setId((int) $qb->getLastInsertId());
```

Comment:
Why is this?
It is completely logical, but why not add it before?

Reply:
We didn't added it before (when we wrote this), because, I guess, we never needed id of inserted face before:D

Reply:
Ok ok..
:sweat_smile: